config/configgrpc/README.md

```diff
@@ -44,6 +44,45 @@ exporters:
       "test 2": "value 2"
 ```
 
+### Compression Comparison
```

Comment:
Please consider to add system details used for this, would probably be good if we can run these benchmarks on a cloud VM which is probably the majority of the time for our users.

Reply:
Yes I can do that. 

Where should we put the docs for this? I put them in `config/configgrpc/README.md `, but that doesn't seem right since there will be similar discussion for the otlp http exporters as well. What about a new file at `docs/compression.md`?